examples/pom.xml

```diff
@@ -38,13 +38,15 @@
     <module>access-timeout-meta</module>
     <module>access-timeout</module>
     <module>alternate-descriptors</module>
+    <module>applet</module>
```

Comment:
Hi @ShermanMarshall  I tried today to build locally this PR and I got the following error related with the inclusion of the applet module.

This what I did on a box with JDK 1.8.0_151-b12, ios:

1. git checkout https://github.com/apache/tomee/pull/448  //I have hub installed
2.git pull upstream master //merging latest master changes into your PR that at this point I have locally on my machine.
3. clean install -U -DskipTests

[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Building OpenEJB :: Web Examples :: Signed Applet EJB Client 8.0.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO]
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ applet ---
[INFO]
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ applet ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /Users/cesar/git/tomee/examples/applet/src/main/resources
[INFO]
[INFO] --- maven-dependency-plugin:2.10:copy (copy) @ applet ---
[INFO] Configured Artifact: org.apache.tomee:openejb-client:8.0.0-SNAPSHOT:jar
[INFO] Configured Artifact: org.apache.tomee:javaee-api:8.0:jar
[INFO] Copying openejb-client-8.0.0-SNAPSHOT.jar to /Users/cesar/git/tomee/examples/applet/target/applet/openejb-client.jar
[INFO] Copying javaee-api-8.0.jar to /Users/cesar/git/tomee/examples/applet/target/applet/javaee-api.jar
[INFO]
[INFO] --- maven-compiler-plugin:3.5.1:compile (default-compile) @ applet ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 3 source files to /Users/cesar/git/tomee/examples/applet/target/classes
[INFO]
[INFO] --- maven-antrun-plugin:1.5:run (default) @ applet ---
[WARNING] Parameter tasks is deprecated, use target instead
[INFO] Executing tasks

main:
      [jar] Building jar: /Users/cesar/git/tomee/examples/applet/target/applet/app.jar
  [signjar] Signing JAR: /Users/cesar/git/tomee/examples/applet/target/applet/app.jar to /Users/cesar/git/tomee/examples/applet/target/applet/app.jar as mykey
  [signjar] jarsigner error: java.lang.RuntimeException: keystore load: /Users/cesar/.keystore (No such file or directory)
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] Apache OpenEJB ..................................... SUCCESS [  1.463 s]
[INFO] OpenEJB :: Container ............................... SUCCESS [  0.026 s]
[INFO] OpenEJB :: Container :: Loader ..................... SUCCESS [  4.181 s]
[INFO] OpenEJB :: Container :: MBean Annotation API ....... SUCCESS [  0.206 s]
[INFO] OpenEJB :: Container :: JPA Integration ............ SUCCESS [  0.995 s]
[INFO] OpenEJB :: Container :: API ........................ SUCCESS [  0.467 s]
[INFO] OpenEJB :: Container :: Java Agent ................. SUCCESS [  0.265 s]
[INFO] OpenEJB :: Container :: Java EE .................... SUCCESS [ 11.763 s]
[INFO] OpenEJB :: Container :: Java EE Accessors .......... SUCCESS [  5.486 s]
[INFO] OpenEJB :: iTests .................................. SUCCESS [  0.028 s]
[INFO] OpenEJB :: iTests :: Beans ......................... SUCCESS [  4.117 s]
[INFO] OpenEJB :: iTests :: Interceptor Beans ............. SUCCESS [  0.316 s]
[INFO] OpenEJB :: iTests :: Servlets ...................... SUCCESS [  0.452 s]
[INFO] OpenEJB :: iTests :: Client ........................ SUCCESS [  4.951 s]
[INFO] OpenEJB :: iTests :: App ........................... SUCCESS [  0.799 s]
[INFO] OpenEJB :: iTests :: Failover EJB .................. SUCCESS [  0.162 s]
[INFO] OpenEJB :: Container :: Core ....................... SUCCESS [02:33 min]
[INFO] OpenEJB :: Server .................................. SUCCESS [  0.027 s]
[INFO] OpenEJB :: Server :: Client ........................ SUCCESS [  2.635 s]
[INFO] OpenEJB :: Server :: Core .......................... SUCCESS [  1.988 s]
[INFO] OpenEJB :: Server :: Multicast Discovery ........... SUCCESS [  1.387 s]
[INFO] OpenEJB :: Server :: EJBd .......................... SUCCESS [  2.243 s]
[INFO] OpenEJB :: Server :: Hsql .......................... SUCCESS [  0.938 s]
[INFO] OpenEJB :: Server :: Http .......................... SUCCESS [  2.050 s]
[INFO] OpenEJB :: Server :: Webservices ................... SUCCESS [  1.590 s]
[INFO] OpenEJB :: Server :: CXF Transport ................. SUCCESS [  1.539 s]
[INFO] OpenEJB :: Server :: CXF ........................... SUCCESS [  3.467 s]
[INFO] OpenEJB :: Server :: REST .......................... SUCCESS [  1.106 s]
[INFO] OpenEJB :: Server :: CXF RS ........................ SUCCESS [  3.581 s]
[INFO] OpenEJB :: Server :: Daemon ........................ SUCCESS [  0.793 s]
[INFO] OpenEJB :: Assembly ................................ SUCCESS [  0.019 s]
[INFO] OpenEJB :: Assembly :: Standalone .................. SUCCESS [  8.645 s]
[INFO] OpenEJB :: Arquillian Adaptors Parent .............. SUCCESS [  0.027 s]
[INFO] OpenEJB :: Arquillian Adaptors Parent :: Common .... SUCCESS [  1.154 s]
[INFO] OpenEJB :: TomEE ................................... SUCCESS [  0.542 s]
[INFO] OpenEJB :: TomEE :: Util ........................... SUCCESS [  0.612 s]
[INFO] OpenEJB :: Arquillian Adaptors Parent :: TomEE Common SUCCESS [  1.389 s]
[INFO] OpenEJB :: iTests :: Failover ...................... SUCCESS [  1.513 s]
[INFO] OpenEJB :: iTests :: Legacy Client ................. SUCCESS [  2.174 s]
[INFO] OpenEJB :: iTests :: Legacy Server ................. SUCCESS [  1.442 s]
[INFO] OpenEJB :: iTests :: Web ........................... SUCCESS [  0.440 s]
[INFO] OpenEJB :: TomEE :: Juli ........................... SUCCESS [  2.768 s]
[INFO] OpenEJB :: Maven Plugins ........................... SUCCESS [  0.324 s]
[INFO] OpenEJB :: Maven Plugins :: Util ................... SUCCESS [  1.136 s]
[INFO] OpenEJB :: TomEE :: Common ......................... SUCCESS [  1.175 s]
[INFO] OpenEJB :: TomEE :: Config ......................... SUCCESS [  1.199 s]
[INFO] OpenEJB :: TomEE :: Loader ......................... SUCCESS [  2.376 s]
[INFO] OpenEJB :: TomEE :: JDBC ........................... SUCCESS [  0.932 s]
[INFO] OpenEJB :: TomEE :: Catalina ....................... SUCCESS [  4.810 s]
[INFO] OpenEJB :: TomEE :: MyFaces ........................ SUCCESS [  1.169 s]
[INFO] OpenEJB :: TomEE :: Mojarra ........................ SUCCESS [  2.373 s]
[INFO] OpenEJB :: TomEE :: RS Web Services ................ SUCCESS [  5.093 s]
[INFO] OpenEJB :: TomEE :: Webapp ......................... SUCCESS [ 10.803 s]
[INFO] OpenEJB :: TomEE :: Web Services ................... SUCCESS [  4.112 s]
[INFO] OpenEJB :: Microprofile JWT ........................ SUCCESS [  3.135 s]
[INFO] OpenEJB :: TomEE :: MicroProfile ................... SUCCESS [  0.654 s]
[INFO] OpenEJB :: TomEE :: MicroProfile Common ............ SUCCESS [  2.965 s]
[INFO] OpenEJB :: TomEE :: Plus Webapp .................... SUCCESS [  9.347 s]
[INFO] OpenEJB :: Utils ................................... SUCCESS [  0.015 s]
[INFO] OpenEJB :: Utils :: Core EclipseLink ............... SUCCESS [  0.689 s]
[INFO] OpenEJB :: TomEE :: Plume Webapp ................... SUCCESS [  9.018 s]
[INFO] OpenEJB :: TomEE :: MicroProfile Webapp ............ SUCCESS [01:06 min]
[INFO] OpenEJB :: TomEE :: Apache TomEE ................... SUCCESS [ 37.534 s]
[INFO] OpenEJB :: Maven Plugins :: TomEE Maven Plugin ..... SUCCESS [  7.356 s]
[INFO] OpenEJB :: TomEE :: TomEE Embedded ................. SUCCESS [ 20.132 s]
[INFO] OpenEJB :: Utils :: LiveReload ..................... SUCCESS [  0.375 s]
[INFO] OpenEJB :: Maven Plugins :: TomEE Embedded Maven Plugin SUCCESS [ 10.698 s]
[INFO] OpenEJB :: Maven Plugins :: OpenEJB Embedded Maven Plugin SUCCESS [  3.848 s]
[INFO] OpenEJB :: Maven Plugins :: TomEE Webapp Archetype . SUCCESS [  0.147 s]
[INFO] OpenEJB :: Maven Plugins :: jars.txt Maven Plugin .. SUCCESS [  0.475 s]
[INFO] OpenEJB :: Maven Plugins :: ApplicationComposer Maven Plugin SUCCESS [  4.727 s]
[INFO] OpenEJB :: Gradle Plugins .......................... SUCCESS [  0.047 s]
[INFO] OpenEJB :: Gradle Plugins :: TomEE Embedded ........ SUCCESS [  0.356 s]
[INFO] OpenEJB :: Container :: JUnit ...................... SUCCESS [  0.836 s]
[INFO] OpenEJB :: Server :: ActiveMQ ...................... SUCCESS [  0.558 s]
[INFO] OpenEJB :: Server :: Derby Network Service ......... SUCCESS [  0.576 s]
[INFO] OpenEJB :: Server :: Axis .......................... SUCCESS [  1.445 s]
[INFO] OpenEJB :: Server :: Common CLI .................... SUCCESS [  0.841 s]
[INFO] OpenEJB :: Server :: SSH ........................... SUCCESS [  0.833 s]
[INFO] OpenEJB :: Server :: BoneCP ........................ SUCCESS [  0.277 s]
[INFO] OpenEJB :: Server :: Hessian ....................... SUCCESS [  0.534 s]
[INFO] OpenEJB :: Examples :: @AccessTimeout (Meta) ....... SUCCESS [  1.001 s]
[INFO] OpenEJB :: Examples :: @AccessTimeout .............. SUCCESS [  0.092 s]
[INFO] OpenEJB :: Examples :: Alternate Descriptors ....... SUCCESS [  0.136 s]
[INFO] OpenEJB :: Web Examples :: Signed Applet EJB Client  FAILURE [  2.223 s]
[INFO] OpenEJB :: Examples :: Application Composer ........ SKIPPED
[INFO] OpenEJB :: Arquillian Adaptors Parent :: ZipLock ... SKIPPED
[INFO] OpenEJB :: Examples :: Application Composer, JAX-WS and CDI are in a boat SKIPPED
[INFO] OpenEJB :: Examples :: @ApplicationException inheritance SKIPPED
[INFO] OpenEJB :: Arquillian Adaptors Parent :: Transaction Provider SKIPPED
[INFO] OpenEJB :: Arquillian Adaptors Parent :: Remote .... SKIPPED
[INFO] OpenEJB :: Examples :: Arquillian Persistence Extension Sample SKIPPED
[INFO] OpenEJB :: Examples :: @Asynchronous Methods ....... SKIPPED
[INFO] OpenEJB :: Examples :: @Asynchronous @PostConstrct . SKIPPED
[INFO] OpenEJB :: Examples :: Bean Validation Design By Contract SKIPPED
[INFO] bval-evaluation-redeployment ....................... SKIPPED
[INFO] WebApp1 ............................................ SKIPPED
[INFO] WebApp2 ............................................ SKIPPED
[INFO] runner ............................................. SKIPPED
[INFO] OpenEJB :: Examples :: CDI Stereotypes ............. SKIPPED
[INFO] OpenEJB :: Examples :: CDI Application Scope ....... SKIPPED
[INFO] OpenEJB :: Examples :: Basic CDI ................... SKIPPED
[INFO] OpenEJB :: Examples :: CDI, EJBContext and JAAS .... SKIPPED
[INFO] OpenEJB :: Examples :: CDI Events .................. SKIPPED
[INFO] OpenEJB :: Examples :: CDI Interceptors ............ SKIPPED
[INFO] OpenEJB :: Examples :: CDI-Disposes ................ SKIPPED
[INFO] OpenEJB :: Examples :: CDI-Field Producer .......... SKIPPED
[INFO] OpenEJB :: Examples :: Basic Qualifier ............. SKIPPED
[INFO] OpenEJB :: Examples :: CDI Realm ................... SKIPPED
[INFO] OpenEJB :: Examples :: CDI Request Scope ........... SKIPPED
[INFO] OpenEJB :: Examples :: CDI Session Scope ........... SKIPPED
[INFO] OpenEJB :: Arquillian Adaptors Parent :: Embedded Adaptor SKIPPED
[INFO] OpenEJB :: Web Examples :: Change JAXWS URL ........ SKIPPED
[INFO] OpenEJB :: Examples :: EJB 2.1 Component Interfaces  SKIPPED
[INFO] OpenEJB :: Examples :: Client Resource Lookup ...... SKIPPED
[INFO] OpenEJB :: Connector Examples :: Connector in EAR .. SKIPPED
[INFO] OpenEJB :: Connector Examples :: Connector in EAR :: API SKIPPED
[INFO] OpenEJB :: Connector Examples :: Connector in EAR :: Connector Impl SKIPPED
[INFO] OpenEJB :: Connector Examples :: Connector in EAR :: RAR Packaging SKIPPED
[INFO] OpenEJB :: Connector Examples :: Connector in EAR :: Sample WAR SKIPPED
[INFO] OpenEJB :: Connector Examples :: Connector in EAR :: EAR Packaging SKIPPED
[INFO] OpenEJB :: Connector Examples :: Connector in EAR :: Functional Tests SKIPPED
[INFO] OpenEJB :: Connector Examples :: Connector in WAR .. SKIPPED
[INFO] OpenEJB :: Examples :: Cucumber JVM ................ SKIPPED
[INFO] OpenEJB :: Examples :: Expanded support for Env Entries SKIPPED
[INFO] OpenEJB :: Examples :: Datasource Ciphered Password  SKIPPED
[INFO] OpenEJB :: Examples :: Datasource Definition ....... SKIPPED
[INFO] OpenEJB :: Examples :: Datasource Versioning ....... SKIPPED
[INFO] OpenEJB :: Examples :: Decorators .................. SKIPPED
[INFO] OpenEJB :: Arquillian Adaptors Parent :: OpenEJB Container SKIPPED
[INFO] OpenEJB :: Examples :: DeltaSpike @ConfigProperty .. SKIPPED
[INFO] OpenEJB :: Examples :: DeltaSpike Exception Handling SKIPPED
[INFO] OpenEJB :: Examples :: JSF2/CDI/BV/JPA/DeltaSpike .. SKIPPED
[INFO] OpenEJB :: Examples :: DeltaSpike I18n ............. SKIPPED
[INFO] OpenEJB :: Examples :: Dynamic DAO Implementation .. SKIPPED
[INFO] OpenEJB :: Examples :: Dynamic Datasource Routing .. SKIPPED
[INFO] OpenEJB :: Examples :: Dynamic Implementation ...... SKIPPED
[INFO] OpenEJB :: Examples :: Dynamic MBean Proxy ......... SKIPPED
[INFO] OpenEJB :: Examples :: Ear Testing ................. SKIPPED
[INFO] OpenEJB :: Examples :: Ear Testing :: Business Model SKIPPED
[INFO] OpenEJB :: Examples :: Ear Testing :: Business Logic SKIPPED
[INFO] OpenEJB :: Web Examples :: EJB Examples War ........ SKIPPED
[INFO] OpenEJB :: Web Examples :: EJB Remote Call ......... SKIPPED
[INFO] OpenEJB :: Web Examples :: EJB Remote Call ......... SKIPPED
[INFO] OpenEJB :: Web Examples :: EJB WebService .......... SKIPPED
[INFO] OpenEJB :: Examples :: Groovy CDI .................. SKIPPED
[INFO] OpenEJB :: Examples :: Groovy JPA .................. SKIPPED
[INFO] OpenEJB :: Examples :: Groovy Spock ................ SKIPPED
[INFO] OpenEJB :: Examples :: Hello World - Weblogic ...... SKIPPED
[INFO] OpenEJB :: Examples :: @Resource javax.jms.ConnectionFactory SKIPPED
[INFO] OpenEJB :: Examples :: @Resource DataSource Injection SKIPPED
[INFO] OpenEJB :: Examples :: @EJB Injection .............. SKIPPED
[INFO] OpenEJB :: Examples :: @PersistenceContext EntityManager Injection SKIPPED
[INFO] OpenEJB :: Examples :: @Resource env-entry Injection SKIPPED
[INFO] OpenEJB :: Examples :: Interceptors ................ SKIPPED
[INFO] OpenEJB :: Examples :: JavaMail API ................ SKIPPED
[INFO] OpenEJB :: Web Examples :: Java Modules (Simple REST resource) SKIPPED
[INFO] jaxrs-filter Maven Webapp .......................... SKIPPED
[INFO] OpenEJB :: Examples :: JPA with EclipseLink ........ SKIPPED
[INFO] OpenEJB :: Utils :: Core Hibernate ................. SKIPPED
[INFO] OpenEJB :: Examples :: JPA with Hibernate .......... SKIPPED
[INFO] OpenEJB :: Examples :: JPA @Enumerated ............. SKIPPED
[INFO] OpenEJB :: Web Examples :: JSF - ManangedBean and EJB SKIPPED
[INFO] OpenEJB :: Web Examples :: JSF - CDI and EJB ....... SKIPPED
[INFO] OpenEJB :: Examples :: @EJB Lookup ................. SKIPPED
[INFO] OpenEJB :: Examples :: EJB Lookup with descriptor .. SKIPPED
[INFO] OpenEJB :: Examples :: MBean Auto Registration ..... SKIPPED
[INFO] OpenEJB :: Examples :: Movies Complete (Meta) ...... SKIPPED
[INFO] OpenEJB :: Examples :: Movies Complete ............. SKIPPED
[INFO] MicroProfile :: Examples :: Config ................. SKIPPED
[INFO] MicroProfile :: Examples :: Config Source Database . SKIPPED
[INFO] OpenEJB :: Examples :: Microprofile Fault Tolerance :: Fallback SKIPPED
[INFO] OpenEJB :: Examples :: Microprofile Fault Tolerance :: Retry SKIPPED
[INFO] OpenEJB :: Examples :: Microprofile Metrics Counted  SKIPPED
[INFO] OpenEJB :: Examples :: Microprofile Metrics Histogram SKIPPED
[INFO] OpenEJB :: Examples ................................ SKIPPED
[INFO] mp-metrics-metered ................................. SKIPPED
[INFO] OpenEJB :: Examples :: Microprofile Metrics Timed .. SKIPPED
[INFO] mp-opentracing.traced .............................. SKIPPED
[INFO] OpenEJB :: Examples :: MP REST JWT ................. SKIPPED
[INFO] OpenEJB :: Examples :: MP REST JWT JWK ............. SKIPPED
[INFO] OpenEJB :: Examples :: MP REST JWT PRINCIPAL ....... SKIPPED
[INFO] OpenEJB :: Examples :: MP REST JWT Public Key ...... SKIPPED
[INFO] OpenEJB :: Utils :: Mockito ........................ SKIPPED
[INFO] OpenEJB :: Examples :: MTOM ........................ SKIPPED
[INFO] OpenEJB :: Examples :: Multiple JPA providers ...... SKIPPED
[INFO] OpenEJB :: Examples :: Multiple Arquillian Adapters  SKIPPED
[INFO] OpenEJB :: Examples :: Multiple TomEE with Arquillian SKIPPED
[INFO] OpenEJB :: Examples :: JSF2/CDI/BV/JPA/CODI ........ SKIPPED
[INFO] OpenEJB :: Examples :: Persistence Fragment ........ SKIPPED
[INFO] OpenEJB :: Web Examples :: Pojo WS ................. SKIPPED
[INFO] OpenEJB :: Examples :: Polling ..................... SKIPPED
[INFO] OpenEJB :: Examples :: Polling :: Domain ........... SKIPPED
[INFO] OpenEJB :: Examples :: Polling :: Core ............. SKIPPED
[INFO] OpenEJB :: Examples :: Polling :: Web .............. SKIPPED
[INFO] OpenEJB :: Examples :: Polling :: Client ........... SKIPPED
[INFO] OpenEJB :: Examples :: Polling :: MDB .............. SKIPPED
[INFO] OpenEJB :: Examples :: DeltaSpike ProjectStage ..... SKIPPED
[INFO] OpenEJB :: Examples :: Quartz ...................... SKIPPED
[INFO] OpenEJB :: Examples :: Quartz :: Quartz Resource Adapter SKIPPED
[INFO] OpenEJB :: Examples :: Quartz :: Quartz Beans ...... SKIPPED
[INFO] OpenEJB :: Examples :: DataSource Realm ............ SKIPPED
[INFO] OpenEJB :: Examples :: Reloadable Persistence Unit Properties SKIPPED
[INFO] OpenEJB :: Web Examples :: Resource Declared In A Webapp SKIPPED
[INFO] OpenEJB :: Examples :: REST, Mockito and Application Composer SKIPPED
[INFO] OpenEJB :: Examples :: REST and Application Composer SKIPPED
[INFO] OpenEJB :: Examples :: REST CDI .................... SKIPPED
[INFO] OpenEJB :: Examples :: JAXRS and JAAS .............. SKIPPED
[INFO] OpenEJB :: Examples :: REST and EJB ................ SKIPPED
[INFO] OpenEJB :: Web Examples :: REST Example ............ SKIPPED
[INFO] OpenEJB :: Web Examples :: REST Example With Application SKIPPED
[INFO] OpenEJB :: Examples :: REST XML JSON ............... SKIPPED
[INFO] OpenEJB :: Examples :: Basic Scala ................. SKIPPED
[INFO] OpenEJB :: Examples :: ScheduleExpression .......... SKIPPED
[INFO] OpenEJB :: Examples :: @Schedule Events ............ SKIPPED
[INFO] OpenEJB :: Examples :: @Schedule Methods (Meta) .... SKIPPED
[INFO] OpenEJB :: Examples :: @Schedule Methods ........... SKIPPED
[INFO] OpenEJB :: Examples :: Server Events ............... SKIPPED
[INFO] OpenEJB :: Examples :: Simple CDI Interceptor ...... SKIPPED
[INFO] OpenEJB :: Examples :: Simple EAR .................. SKIPPED
[INFO] OpenEJB :: Examples :: Simple EAR :: Business Model  SKIPPED
[INFO] OpenEJB :: Examples :: Simple EAR :: Business Logic  SKIPPED
[INFO] OpenEJB :: Examples :: Simple EAR :: Webapp ........ SKIPPED
[INFO] OpenEJB :: Examples :: Simple EAR :: EAR Packaging . SKIPPED
[INFO] OpenEJB :: Examples :: Simple EAR :: Functional Tests SKIPPED
[INFO] OpenEJB :: Examples :: Simple CMP2 Entity .......... SKIPPED
[INFO] OpenEJB :: Examples :: Simple MDB With a CDI Injection SKIPPED
[INFO] OpenEJB :: Examples :: Simple MDB Example .......... SKIPPED
[INFO] OpenEJB :: Examples :: Simple MDB Using Deployment Descriptor Example SKIPPED
[INFO] OpenEJB :: Examples :: Simple Remote tomcat-users.xml SKIPPED
[INFO] OpenEJB :: Examples :: Simple REST ................. SKIPPED
[INFO] OpenEJB :: Examples :: Simple Singleton ............ SKIPPED
[INFO] OpenEJB :: Examples :: Simple Stateful Pojo ........ SKIPPED
[INFO] OpenEJB :: Examples :: Simple Stateful Pojo Callbacks SKIPPED
[INFO] OpenEJB :: Examples :: Simple Stateless Pojo ....... SKIPPED
[INFO] OpenEJB :: Examples :: Simple Stateless Pojo Callbacks SKIPPED
[INFO] OpenEJB :: Examples :: Simple Stateless With Deployment Descriptor SKIPPED
[INFO] OpenEJB :: Examples :: Simple Webservice ........... SKIPPED
[INFO] OpenEJB :: Examples :: Simple Webservice Without Interface SKIPPED
[INFO] OpenEJB :: Examples :: Spring Data ................. SKIPPED
[INFO] OpenEJB :: Examples :: Spring Data Meta ............ SKIPPED
[INFO] OpenEJB :: Web Examples :: Struts .................. SKIPPED
[INFO] OpenEJB :: Examples :: Telephone Stateful Pojo ..... SKIPPED
[INFO] OpenEJB :: Examples :: TestCase Injection .......... SKIPPED
[INFO] OpenEJB :: Examples :: Testing Security (Meta) ..... SKIPPED
[INFO] OpenEJB :: Examples :: Testing Security ............ SKIPPED
[INFO] OpenEJB :: Examples :: Testing Security ............ SKIPPED
[INFO] OpenEJB :: Examples :: Testing Security Service Provider SKIPPED
[INFO] OpenEJB :: Examples :: Testing Security Script Service Provider SKIPPED
[INFO] OpenEJB :: Examples :: Testing Transactions ........ SKIPPED
[INFO] OpenEJB :: Examples :: Testing Transactions BMT .... SKIPPED
[INFO] OpenEJB :: Examples :: TomEE, Jersey, Eclipselink .. SKIPPED
[INFO] OpenEJB :: Examples :: Transaction Rollback ........ SKIPPED
[INFO] OpenEJB :: Examples :: Troubleshooting ............. SKIPPED
[INFO] OpenEJB :: Examples :: Webservice Attachments ...... SKIPPED
[INFO] OpenEJB :: Examples :: Webservice Inheritance ...... SKIPPED
[INFO] OpenEJB :: Web Examples :: EJB WebService with Security SKIPPED
[INFO] OpenEJB :: Web Examples :: EJB WebService with WS-Security SKIPPED
[INFO] OpenEJB :: Web Examples :: EJB WebService WS Security with resources.xml SKIPPED
[INFO] OpenEJB :: Examples :: Web Service Handlers ........ SKIPPED
[INFO] OpenEJB :: Examples :: @WebService Holder .......... SKIPPED
[INFO] OpenEJB :: Web Examples :: Moviefun ................ SKIPPED
[INFO] OpenEJB :: Web Examples :: Moviefun Rest ........... SKIPPED
[INFO] OpenEJB :: Utils :: Provisionning .................. SKIPPED
[INFO] OpenEJB :: Examples :: Resources/JMX Example ....... SKIPPED
[INFO] OpenEJB :: Examples :: Resources/JMX Example :: EJB Module SKIPPED
[INFO] OpenEJB :: Examples :: Resources/JMX Example :: EAR Module SKIPPED
[INFO] OpenEJB :: Examples :: WebSocket with tls and basic auth Example SKIPPED
[INFO] OpenEJB :: Examples :: Concurrency utilities example SKIPPED
[INFO] OpenEJB :: Examples :: MVC (CXF-based) ............. SKIPPED
[INFO] OpenEJB :: Examples :: MVC (CXF-based) with Hibernate SKIPPED
[INFO] OpenEJB :: Examples :: Microprofile REST Client .... SKIPPED
[INFO] OpenEJB :: Examples :: MVC (RestEasy-based) ........ SKIPPED
[INFO] OpenEJB :: Examples :: Custom HealthCheck .......... SKIPPED
[INFO] OpenEJB :: Examples :: Microprofile JSONB Configuration SKIPPED
[INFO] OpenEJB :: Examples :: Microprofile JSONB Custom Serializer/Deserializer SKIPPED
[INFO] OpenEJB :: Examples :: Vaadin LTS V08 .............. SKIPPED
[INFO] OpenEJB :: Examples :: Vaadin LTS V10 .............. SKIPPED
[INFO] OpenEJB :: Examples :: Vaadin LATEST ............... SKIPPED
[INFO] OpenEJB :: Examples :: Azure deployment example .... SKIPPED
[INFO] OpenEJB :: Examples :: Microprofile Fault Tolerance :: Timeout SKIPPED
[INFO] OpenEJB :: Assembly :: All In One EJB Lite jar ..... SKIPPED
[INFO] OpenEJB :: Utils :: TomEE Webapps-based Deployer EJB SKIPPED
[INFO] OpenEJB :: TCK ..................................... SKIPPED
[INFO] OpenEJB :: TCK :: Common ........................... SKIPPED
[INFO] OpenEJB :: TCK :: CDI Embedded ..................... SKIPPED
[INFO] OpenEJB :: TCK :: CDI TomEE ........................ SKIPPED
[INFO] OpenEJB :: TCK :: Bean Validation Embedded ......... SKIPPED
[INFO] OpenEJB :: TCK :: Bean Validation TomEE ............ SKIPPED
[INFO] OpenEJB :: TCK :: MicroProfile ..................... SKIPPED
[INFO] OpenEJB :: TCK :: MicroProfile Config TCK .......... SKIPPED
[INFO] OpenEJB :: TCK :: MicroProfile JWT TCK ............. SKIPPED
[INFO] OpenEJB :: TCK :: MicroProfile Fault Tolerance TCK . SKIPPED
[INFO] OpenEJB :: TCK :: MicroProfile Health TCK .......... SKIPPED
[INFO] OpenEJB :: TCK :: MicroProfile Metrics TCK ......... SKIPPED
[INFO] OpenEJB :: TCK :: MicroProfile Rest Client TCK ..... SKIPPED
[INFO] OpenEJB :: TCK :: MicroProfile Open API TCK ........ SKIPPED
[INFO] OpenEJB :: TCK :: MicroProfile Open Tracing TCK .... SKIPPED
[INFO] OpenEJB :: Arquillian Adaptors Parent :: Webapp Remote SKIPPED
[INFO] OpenEJB :: Arquillian Adaptors Parent :: Tests ..... SKIPPED
[INFO] OpenEJB :: Arquillian Adaptors Parent :: Tests :: Web Profile SKIPPED
[INFO] OpenEJB :: Arquillian Adaptors Parent :: Tests :: JAXRS SKIPPED
[INFO] OpenEJB :: Arquillian Adaptors Parent :: Tests :: JAXWS SKIPPED
[INFO] OpenEJB :: Arquillian Adaptors Parent :: Tests :: JMS SKIPPED
[INFO] OpenEJB :: Arquillian Adaptors Parent :: Tests :: CODI SKIPPED
[INFO] OpenEJB :: Arquillian Adaptors Parent :: Tests :: Configuration Tests SKIPPED
[INFO] OpenEJB :: Arquillian Adaptors Parent :: Sample :: Moviefun SKIPPED
[INFO] OpenEJB :: Arquillian Adaptors Parent :: Arquillian TCK SKIPPED
[INFO] OpenEJB :: Utils :: Log4j2 ......................... SKIPPED
[INFO] OpenEJB :: TomEE :: webaccess ...................... SKIPPED
[INFO] OpenEJB :: TomEE :: Overlay Runner ................. SKIPPED
[INFO] OpenEJB :: TomEE :: Security ....................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 07:40 min
[INFO] Finished at: 2019-04-16T11:45:01-06:00
[INFO] Final Memory: 310M/3780M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-antrun-plugin:1.5:run (default) on project applet: An Ant BuildException has occured: jarsigner returned: 1 -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :applet

Reply:
> main:
> [jar] Building jar: /Users/cesar/git/tomee/examples/applet/target/applet/app.jar
> [signjar] Signing JAR: /Users/cesar/git/tomee/examples/applet/target/applet/app.jar to /Users/cesar/git/tomee/examples/applet/target/applet/app.jar as mykey
> [signjar] jarsigner error: java.lang.RuntimeException: keystore load: /Users/cesar/.keystore (No such file or directory)
> [INFO] ------------------------------------------------------------------------
> [INFO] Reactor Summary:
===================================================================

You have to create the keystore first. Instructions are in README.txt

Reply:
Yes, thanks for the pointer @ShermanMarshall  

My observation arises from the fact that, as far as I know, the [buildbot](https://ci.apache.org/builders/tomee-trunk-ubuntu-jvm8) infrastructure is unaware of pre-configurations like keystores or keys.

Among other steps, buildbot first integrate skipping the test and then triggering all the tests (including the ones from the examples). With this PR both commands will fail with the error showed above.

```
mvn -e --show-version --fail-at-end -Djava.io.tmpdir=/tmp/tomee-trunk-ubuntu-jvm8 clean install -DfailIfNoTests=false -DskipTests

mvn --show-version --fail-at-end -Djava.io.tmpdir=/tmp/tomee-trunk-ubuntu-jvm8 clean install -Pall-adapters -Dsurefire.useFile=false -DdisableXmlReport=true -Dopenejb.arquillian.debug=true -DTOMEE_LOCK_FILE=/tmp/tomee.port.lock
```

I'm going to ask for feedback on the mailing list to double check with committers if my analysis is correct or not :).